cmd/ssh-to-age/main.go

```diff
@@ -63,9 +64,12 @@ func convertKeys(args []string) error {
 		}
 		defer writer.Close()
 	}
-
 	if opts.privateKey {
```

Comment:
Thanks for the addition would it be possible to read the password using a function like this?

https://github.com/sapcc/nova-password/blob/6b3e3cf9951797da7d9dcf059245e063d23b6c41/main.go#L368

Than the password does not end up as likely in the the users shell history or a process that steals the secret from cmdline from `/proc`?

Reply:
I think `-password` would become a bool flag in this case.

Reply:
Yeah can do, I've been using `read -s` for this so far but it's a reasonable change. Does it even need to be a bool flag in that case? Surely we could just check if the env var is set and if it is, use it as the password.

Reply:
That also works.